exam_marker/Dockerfile

```diff
@@ -1,5 +1,28 @@
-FROM golang:1.17
-MAINTAINER Christian Heusel <christian@heusel.eu>
```

Comment:
Please re-add the `MAINTAINER` Statement

Reply:
Changed for appropriate Label statement